components/views/admin/dashboard.tsx

```diff
@@ -1,29 +1,83 @@
-import { GrowthChart } from "@/components/shared/GrowthChart";
-import { MentorsChart } from "@/components/shared/MentorsChart";
-import { StatsCards } from "@/components/shared/StatCard";
-import { UsersTable } from "@/components/shared/UserTable";
+"use client"
+
+import { GrowthChart } from "@/components/shared/admin/dashboard/GrowthChart";
+import { MentorsChart } from "@/components/shared/admin/dashboard/MentorsChart";
+import { StatsCards } from "@/components/shared/admin/dashboard/StatCard";
+import { UsersTable } from "@/components/shared/admin/dashboard/UserTable";
+import { endPoints } from "@/data/end-points";
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
+const BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL;
```

Comment:
💡 can you put this in one file and import it. 
like the way you did for router.
we have to manage from one place.